proto/src/determined/api/v1/template.proto

```diff
@@ -9,13 +9,31 @@ import "determined/template/v1/template.proto";
 
 // Get a list of templates.
 message GetTemplatesRequest {
+    // Sorts templates by the given field.
+    enum SortBy {
+        // Returns experiments in an unsorted list.
```

Comment:
non-blocking: replace experiments with templates

Reply:
I actually can't push to the source branch to fix this. merge this as is?

Reply:
I'd rather you close this PR, pull the source branch, add the fixes, open a new PR and link it to this one for context.